examples/lujain_work_area/Contact_Book/contact_records.py

```diff
@@ -0,0 +1,47 @@
+# contact_records.py
+
+import csv
+import os
+
+class ContactRecords:
+    def __init__(self, csv_file):
+        self.csv_file = csv_file
+        self.fieldnames = ['Name', 'Phone', 'Email', 'Address']
+        self.contacts = self.load_contacts()
+
+    def load_contacts(self):
+        if not os.path.exists(self.csv_file):
+            return []
+        with open(self.csv_file, mode='r') as file:
+            reader = csv.DictReader(file)
+            return [row for row in reader]
+
+    def save_contacts(self):
+        with open(self.csv_file, mode='w', newline='') as file:
+            writer = csv.DictWriter(file, fieldnames=self.fieldnames)
+            writer.writeheader()
+            writer.writerows(self.contacts)
+
+    def add_contact(self, contact):
```

Comment:
I think, ContactRecord can be considered as an individual object which represents only a single record. So we might have to refactor a lot of code to make it understandable.